### ISSUE TITLE
Pallet Unit Tests concerning Balances: Modify mock.rs to have ED > 1, to test balances less than ED.

### ISSUE THREAD

**Issue by mattheworris** [LibertyDSNP/frequency#1852]
As a developer, I would like to create some unit tests for situations when the account balance is less than the Existential Deposit (ED).

Currently, the mocks has ED == 1, so it is not possible to test any situations where ED < 1, which some valid edge cases do exist.

- [ ] Modify mock.rs to have ED > 1
- [ ] Modify unit tests that are affected by this change.

**Comment by Aideepakchaudhary** [comment]
@mattheworris Can you please elaborate this issue? like for which pallet you need to update. In todo, i think the condition should be ED < 1.

**Comment by mattheworris** [comment]
@Aideepakchaudhary 
Yes, this issue concerns the `capacity` and `time-release` pallets which use the `fungible` trait to freeze tokens. The issue was created while working on #1779, #1818, and #1873.

[Here](https://github.com/frequency-chain/frequency/blob/main/pallets/time-release/src/mock.rs#L47), the Existential Deposit (ED) for ‎`Test::ExistentialDeposit` is set to `1`. `mock.rs` defines `Test` as a simplified version of the pallet used for unit testing. 

On testnet/mainnet, [ED](https://github.com/frequency-chain/frequency/blob/main/runtime/common/src/constants.rs#L43) is set to a larger number = [1_000_000](https://github.com/frequency-chain/frequency/blob/main/runtime/common/src/constants.rs#L47).

Therefore, there are valid scenarios on testnet/mainnet that can result in an account balance that is below ED but greater than 0. 

Currently, it is not possible to use `mock.rs` and `Test::ExistentialDeposit` to test this scenario, as the only value less than ED in `mock.rs` is 0.

If `Test::ExistentialDeposit` is changed to a larger value, e.g. 100, it would allow unit tests to create a scenario where `0 < account_balance < ED`. 

However, this may also break the logic for the existing tests with ED == 1 and those tests would need to be updated to use the larger ED value.
